### PR TITLE
implement `getParsedAccountInfo` rpc method

### PR DESCRIPTION
**Pull request by tpompon**

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -2945,16 +2945,6 @@
   resolved "https://registry.yarnpkg.com/@metamask/safe-event-emitter/-/safe-event-emitter-2.0.0.tgz#af577b477c683fad17c619a78208cede06f9605c"
   integrity sha512-/kSXhY692qiV1MXu6EeOZvg5nECLclxNXcKCxJ3cXQgYuRymRHpdx/t7JXfsK+JLjwA1e1c1/SBrlQYpusC29Q==
 
-"@miniflare/cache@2.5.1":
```
Comment (armaniferrante) [inline yarn.lock:2948]: Please remove these yarn.lock changes. As a blanket policy, we're not accepting changes to this file.

Comment (tpompon) [reply yarn.lock]: Sure, I just removed the changes on `yarn.lock`

File: packages/background/src/frontend/solana-connection.ts [hunk 2]
```diff
@@ -391,6 +394,17 @@ async function handleGetTokenLargestAccounts(
...
 }
 
+async function handleGetParsedAccountInfo(
+  ctx: Context<SolanaConnectionBackend>,
+  publicKey: string,
+  commitment?: Commitment
+) {
+  const resp = await ctx.backend.getParsedAccountInfo(
+    new PublicKey(publicKey),
+    commitment
```
Comment (armaniferrante) [inline packages/background/src/frontend/solana-connection.ts:404]:
```suggestion
    commitment
 );
```

**Comment by armaniferrante** [comment]
Fixed merge conflicts.

**Pull request merged**